stratum/hal/lib/common/common.proto

```diff
@@ -276,6 +276,12 @@ message PortConfigParams {
     DEVICE_TYPE_VIRTIO_NET = 1;
     DEVICE_TYPE_VIRTIO_BLK = 2;
   }
+  // Packet Direction types
+  enum PacketDirection {
+    DIRECTION_HOST = 0;
+    DIRECTION_NETWORK = 1;
+    DIRECTION_NONE = 2;
```

Comment:
Nit: more self-documenting names, to avoid ambiguity on the actual direction
```suggestion
    DIRECTION_FROM_HOST = 0;
    DIRECTION_FROM_NETWORK = 1;
    DIRECTION_UNKNOWN = 2;
```